packages/file-type-icons/tsconfig.json

```diff
@@ -12,7 +12,10 @@
     "forceConsistentCasingInFileNames": true,
     "moduleResolution": "node",
     "preserveConstEnums": false,
-    "types": []
+    "typeRoots": ["node_modules/@types", "./typings"],
```

Comment:
Should be able to remove this now that you don't have the custom typing

Reply:
removed